project_transportonly/project.json5

```diff
@@ -0,0 +1,340 @@
+                display_name: 'Scenario',
+                description: 'DECARB 2023 High-Resolution Demand Scenarios; DECARB High, with two charging behaviors: unmanaged and overnight slow',
+                file: 'dimensions/scenarios.csv',
+                module: 'dsgrid.dimension.standard',
+            },
+            {
+                type: 'time',
+                class: 'Time',
+                frequency: 'P0DT1H0M0.000000S',
+                name: 'Time-2018-EST-hourly-periodBeginning-noLeapDayAdjustment-total',
+                display_name: 'Time EST',
+                time_type: 'datetime',
+                leap_day_adjustment: 'none',
+                description: 'Time dimension, 2018 hourly EST, hour-beginning, no Leap Day Adjustment, total value',
+                time_interval_type: 'period_beginning',
+                str_format: '%Y-%m-%d %H:%M:%S',
+                timezone: 'EasternStandard',
+                measurement_type: 'total',
+                ranges: [
+                    {
+                        start: '2018-01-01 00:00:00',
+                        end: '2018-12-31 23:00:00',
+                    },
+                ],
+            },
+            {
+                type: 'weather_year',
+                class: 'WeatherYear',
+                name: '2018 Weather Year',
+                display_name: 'Weather 2018',
+                description: 'Weather year, 2018 only',
+                file: '../project/dimensions/weather_years.csv',
+                module: 'dsgrid.dimension.standard',
+            },
+        ],
+        subset_dimensions: [
+            {
+                name: 'End Uses by Sector',
+                display_name: 'end_uses_by_sector',
+                description: 'Selects end uses by sector',
+                type: 'metric',
+                file: 'dimensions/subset/end_uses_by_sector.csv',
+                create_supplemental_dimension: false,
+                selectors: [
+                    {
+                        name: 'transportation_end_uses',
+                        description: 'Transportation End Uses',
+                        column_values: {'fuel_id': 'electricity', 'unit': 'MWh'},
+                    }
+                ]
+            },
+            {
+                name: 'Subsectors by Sector',
+                display_name: 'subsectors_by_sector',
+                description: 'Selects subsectors by sector.',
+                type: 'subsector',
+                file: 'dimensions/subset/subsectors_by_sector.csv',
+                create_supplemental_dimension: false,
+                selectors: [
+                    {
+                        name: 'transportation_subsectors',
+                        description: 'Transportation Subsectors',
+                    }
+                ]
+            }
+        ],
+        supplemental_dimensions: [
+            {
+                type: 'subsector',
+				class: 'Simplified Vehicle Type',
```

Comment:
This causes an error because the class doesn’t exist. It should probably just be `Subsector`.